pom.xml

```diff
@@ -26,13 +26,13 @@
     <properties>
         <revision>1.45</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
         <java.level>8</java.level>
-        <workflow-api-plugin.version>2.30</workflow-api-plugin.version>
         <useBeta>true</useBeta>
-        <workflow-cps-plugin.version>2.32</workflow-cps-plugin.version>
-        <git-plugin.version>3.0.5</git-plugin.version>
-        <workflow-scm-step.version>2.4</workflow-scm-step.version>
+        <maven-plugin.version>3.1.2</maven-plugin.version>
+        <!-- versions for test dependencies -->
+        <compress-artifacts.version>1.10</compress-artifacts.version>
+        <jenkins-test-harness-tools.version>2.0</jenkins-test-harness-tools.version>
```

Comment:
Could just inline. Generally I only recommend using a POM property for a dependency version in cases where two classifiers (e.g., main and `tests`) of the same artifact are needed.